swap_database.yml

```diff
@@ -82,10 +82,12 @@
   hosts: replicant
   tasks:
     - name: copy database from master to replicant
+      become: yes
       become_user: postgres
       command: "pg_basebackup --write-recovery-conf --pgdata=/var/lib/postgresql/{{ postgres_version }}/main --host={{ hostvars[groups.database[0]].ansible_default_ipv4.address }} --port=5432 --username={{ replicator_db_user|default(default_replicator_db_user) }} --xlog-method=stream --checkpoint=fast"
     # Touch recovery.conf to prevent data removal during postgres_replicant's role execution.
     - name: touch recovery.conf
+      become: yes
       file:
         path: "/var/lib/postgresql/{{ postgres_version }}/main/recovery.conf"
         owner: postgres
```

Comment:
Line 94 has a typo:

        grouap: postgres
 should be
        group: postgres